### PR TITLE
resolve vertical branch conflicts

### PR DESCRIPTION
**Pull request by pchenthx1**
Conflicts against Horizontal manifest baseline of WW06 release

Tracked-On: OAM-105873

### CODE REVIEW

File: bsp_diff/caas/vendor/intel/external/onevpl-intel-gpu/0002-onevpl-driver-changes-for-rpl-s-platform.patch [hunk 1]
```diff
@@ -1,141 +0,0 @@
-From 3705a4970e27281f7f379ed8a0509dc5223b5ddb Mon Sep 17 00:00:00 2001
```
Comment (pchenthx1) [inline bsp_diff/caas/vendor/intel/external/onevpl-intel-gpu/0002-onevpl-driver-changes-for-rpl-s-platform.patch:1]: This patch is nullifed by 0003-Revert-onevpl-driver-changes-to-support-rpl-s-platfo.patch
so removed both of them

File: bsp_diff/caas/vendor/intel/mediasdk_c2/0007-MediaSDK_C2-refactor-regression.patch [hunk 2]
```diff
@@ -1,66 +0,0 @@
-From 27b38bb71bd24288ee55f94daf12328df4911d12 Mon Sep 17 00:00:00 2001
```
Comment (pchenthx1) [inline bsp_diff/caas/vendor/intel/mediasdk_c2/0007-MediaSDK_C2-refactor-regression.patch:1]: This patch is already part of horizontal WW06

File: bsp_diff/caas/vendor/intel/mediasdk_c2/0008-Fixed-hevc-testValidateProfileLevel-5-failures.patch [hunk 3]
```diff
@@ -1,130 +0,0 @@
-From a36f9aae5621ae4590c10bc22cbdbcfe04948610 Mon Sep 17 00:00:00 2001
```
Comment (pchenthx1) [inline bsp_diff/caas/vendor/intel/mediasdk_c2/0008-Fixed-hevc-testValidateProfileLevel-5-failures.patch:1]: This patch is already part of horizontal WW06

File: bsp_diff/caas/vendor/intel/mediasdk_c2/0009-Change-the-MinInputBufferSize-to-1M.patch [hunk 4]
```diff
@@ -1,54 +0,0 @@
-From f2b2b18eff3e5e89f5b3560567b750233aa58176 Mon Sep 17 00:00:00 2001
```
Comment (pchenthx1) [inline bsp_diff/caas/vendor/intel/mediasdk_c2/0009-Change-the-MinInputBufferSize-to-1M.patch:1]: This patch is already part of horizontal WW06

File: bsp_diff/caas/vendor/intel/mediasdk_c2/0009-c2-parameter-Fix-hevc-testValidateProfileLevel.patch [hunk 5]
```diff
@@ -1,28 +0,0 @@
-From d88e36f411f82ca6e00edb3babacfc4d8c1b3ead Mon Sep 17 00:00:00 2001
```
Comment (pchenthx1) [inline bsp_diff/caas/vendor/intel/mediasdk_c2/0009-c2-parameter-Fix-hevc-testValidateProfileLevel.patch:1]: This patch is already part of horizontal WW06

**Comment by pchenthx1** [comment]
Reviewers: @basilchew @bnkoppad kindly review

**Review by prashanth-intel — APPROVED** [review]
LGTM.

**Review by bnkoppad — APPROVED** [review]

File: bsp_diff/caas/hardware/intel/external/drm-intel/0001-libdrm-changes-for-rpl-s.patch [hunk 6]
```diff
@@ -26,19 +26,20 @@ index c00ac546..03ad7105 100644
 +	INTEL_VGA_DEVICE(0xA78A, info), \
 +	INTEL_VGA_DEVICE(0xA78B, info)
```
Comment (basilchew) [inline bsp_diff/caas/hardware/intel/external/drm-intel/0001-libdrm-changes-for-rpl-s.patch:28]: There is an overlap in ID between RPL-S and RPL-P:

```
/* RPL-S */
#define INTEL_RPLS_IDS(info) \
        INTEL_VGA_DEVICE(0xA780, info), \
        INTEL_VGA_DEVICE(0xA781, info), \
        INTEL_VGA_DEVICE(0xA782, info), \
        INTEL_VGA_DEVICE(0xA783, info), \
        INTEL_VGA_DEVICE(0xA788, info), \
        INTEL_VGA_DEVICE(0xA789, info), \
        INTEL_VGA_DEVICE(0xA78A, info), \
        INTEL_VGA_DEVICE(0xA78B, info)

/*RPL-P*/
#define INTEL_RPLP_IDS(info) \
       INTEL_VGA_DEVICE(0xA7A0, info), \
       INTEL_VGA_DEVICE(0xA720, info), \
       INTEL_VGA_DEVICE(0xA7A8, info), \
       INTEL_VGA_DEVICE(0xA7A1, info), \
       INTEL_VGA_DEVICE(0xA721, info), \
       INTEL_VGA_DEVICE(0xA7A9, info), \
       INTEL_VGA_DEVICE(0xA780, info), \
       INTEL_VGA_DEVICE(0xA781, info), \
       INTEL_VGA_DEVICE(0xA782, info), \
       INTEL_VGA_DEVICE(0xA783, info), \
       INTEL_VGA_DEVICE(0xA788, info), \
       INTEL_VGA_DEVICE(0xA789, info), \
       INTEL_VGA_DEVICE(0xA78A, info), \
       INTEL_VGA_DEVICE(0xA78B, info)

```

Please check if this is expected.

File: bsp_diff/caas/vendor/intel/external/onevpl-intel-gpu/0004-Adding-ADL-N-support.patch [hunk 7]
```diff
@@ -117,16 +104,22 @@ index 07ff71b..33cc609 100755
...
 +    { 0x46D1, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
 +    { 0x46D2, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
-+    
++
+     /*RPL-P*/
+     { 0xA7A0, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
+     { 0xA720, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
+@@ -463,6 +470,22 @@ typedef struct {
+     { 0xA721, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
+     { 0xA7A9, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
```
Comment (basilchew) [inline bsp_diff/caas/vendor/intel/external/onevpl-intel-gpu/0004-Adding-ADL-N-support.patch:121]: There are duplicates for RPL-P:

```
   /*RPL-P*/
    { 0xA7A0, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA720, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A8, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A1, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA721, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A9, MFX_HW_ADL_P, MFX_GT2 },//RPL-P

    /* ADL-N */
    { 0x46D0, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
    { 0x46D1, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
    { 0x46D2, MFX_HW_ADL_N, MFX_GT1 },//ADL-N

    /*RPL-P*/
    { 0xA7A0, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA720, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A8, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A1, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA721, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A9, MFX_HW_ADL_P, MFX_GT2 },//RPL-P

```

It seems like the commits are duplicated:
```

**d12e46e (HEAD) Add device IDs for RPL-P**
2a17924 Adding ADL-N support.
7e9e0c8 Add missing ADL device IDs
**4259734 (github/celadon/s/mr0/stable) Add device IDs for RPL-P**
97b2b58 Add Android T definitions for onevpl-intel-gpu

```

**Review by basilchew — CHANGES_REQUESTED** [review]

File: bsp_diff/caas/vendor/intel/external/onevpl-intel-gpu/0004-Adding-ADL-N-support.patch (hunk 7 above)
Comment (pchenthx1) [reply bsp_diff/caas/vendor/intel/external/onevpl-intel-gpu/0004-Adding-ADL-N-support.patch:121]: Duplicate commit is removed by https://github.com/projectceladon/vendor-intel-utils-vertical-iot/pull/27

File: bsp_diff/caas/hardware/intel/external/drm-intel/0001-libdrm-changes-for-rpl-s.patch (hunk 6 above)
Comment (pchenthx1) [reply bsp_diff/caas/hardware/intel/external/drm-intel/0001-libdrm-changes-for-rpl-s.patch]: corrected and updated as per new Horizontal PR https://github.com/projectceladon/external-libdrm/pull/30/

**Review by basilchew — APPROVED** [review]

**Pull request merged**